src/main/java/pl/gispartner/ResourceManagerApp/service/UserService.java

```diff
@@ -0,0 +1,34 @@
+package pl.gispartner.ResourceManagerApp.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import pl.gispartner.ResourceManagerApp.model.UserDto;
+import pl.gispartner.ResourceManagerApp.model.UserEntity;
+import pl.gispartner.ResourceManagerApp.model.UserMapper;
+import pl.gispartner.ResourceManagerApp.persistance.UserRepository;
+
+import java.util.Date;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public Long saveUser(UserDto userDto) {
+        UserEntity userEntity = userRepository.save(UserMapper.mapToEntity(userDto));
+        return userEntity.getId();
+    }
+
+    public void deleteUser(Long userId) {
+        userRepository.deleteById(userId);
```

Comment:
te walidacje o ktorym pisalem (czy odpowiedni uzytkownik najlepiej zrobic w serwisie), tutaj np. moglbys zwalidowac ze jesli usuwasz siebie to nie ma problemu, a zeby usunac innego uzytkownika to musisz byc SUPER_USER